src/expressions.rst

```diff
@@ -2460,14 +2457,6 @@ assigned_operand, modifying_operand)`` is invoked.
 For a :t:`subtraction assignment`, ``core::ops::SubAssign::sub_assign(&mut
 assigned_operand, modifying_operand)`` is invoked.
 
-.. rubric:: Undefined Behavior
-
-:dp:`fls_uywamh3nzl6p`
-It is undefined behavior for an :t:`addition assignment`, a :t:`division
```

Comment:
I am not sure that this should be removed, unless these do not cause overflow. Arithmetic expressions use different `core` functions, so we can't argue that `+=` overflows just because it contains `+`.

Reply:
These are by definition not undefined behaviour, if it is a user implementation, it just does what the user has implemented them to do. If this is a builtin implementation (that is arithmetic on primitives), it is also never undefined behaviour, it will either wrap, panic or abort.

Reply:
Understood.